mantidproject/mantid
# Create phase table tab

Merge after ~~#25513~~

**Description of work.**

This adds a phase tab to the Muon frequency domain analysis GUI This tab can be used to calculate phase tables for a given input workspace as well as create a phase quad workspace which is effectively the information from ever detector combined into a single spectrum with phase differences factored out.

**To test:**
 * Review the code
 * Open the frequency domain analysis tab switch to the instrument MUSR and then go to the phase tab.
* Click on calculate phase table followed by calculate phase quad. Check that a phase table and a phasequad workspace appear in the ADS.
* Examine the history of the workspaces and check that the relevant algorithms have been called with the correct options.



Fixes #25511  




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
